build/tutorials/tutorials-contest/NFT-Marketplace-on-Avalanche/README.md

```diff
@@ -0,0 +1,1828 @@
+This function will iterate over all auctions and return a list of auction addresses.
+
+```solidity
+// Return a list of all auctions
+function getAuctions() external view returns(address[] memory _auctions) {
+    _auctions = new address[](_auctionIdCounter); // create an array of size equal to the current value of the counter
+    for(uint i = 0; i < _auctionIdCounter; i++) { // for each auction
+        _auctions[i] = address(auctions[i]); // add the address of the auction to the array
+    }
+    return _auctions; // return the array
+}
+```
+
+Then, we can use this list of auctions to display them on the web page. We will need a function to obtain information about the auctions. Given an array of auction addresses, we would like to get direct buy price, auction creator, starting price, highest bid, token id, auction state, and the end time.
+
+```solidity
+// Return the information of each auction address
+function getAuctionInfo(address[] calldata _auctionsList)
+    external
+    view
+    returns (
+        uint256[] memory directBuy,
+        address[] memory owner,
+        uint256[] memory highestBid,
+        uint256[] memory tokenIds,
+        uint256[] memory endTime,
+        uint256[] memory startPrice,
+        uint256[] memory auctionState
+    )
+{
+    directBuy = new uint256[](_auctionsList.length); // create an array of size equal to the length of the passed array
+    owner = new address[](_auctionsList.length); // create an array of size equal to the length of the passed array
+    highestBid = new uint256[](_auctionsList.length);
+    tokenIds = new uint256[](_auctionsList.length);
+    endTime = new uint256[](_auctionsList.length);
+    startPrice = new uint256[](_auctionsList.length);
+    auctionState = new uint256[](_auctionsList.length);
+
+    for (uint256 i = 0; i < _auctionsList.length; i++) { // for each auction
+        directBuy[i] = Auction(auctions[i]).directBuyPrice(); // get the direct buy price
+        owner[i] = Auction(auctions[i]).creator(); // get the owner of the auction
+        highestBid[i] = Auction(auctions[i]).maxBid(); // get the highest bid
+        tokenIds[i] = Auction(auctions[i]).tokenId(); // get the token id
+        endTime[i] = Auction(auctions[i]).endTime(); // get the end time
+        startPrice[i] = Auction(auctions[i]).startPrice(); // get the start price
+        auctionState[i] = uint(Auction(auctions[i]).getAuctionState()); // get the auction state
+    }
+
+    return ( // return the arrays
+        directBuy,
+        owner,
+        highestBid,
+        tokenIds,
+        endTime,
+        startPrice,
+        auctionState
+    );
+}
+```
+
+See the full [AuctionManager.sol here](contracts/AuctionManager.sol)
+
+That's all for the contracts!
+
+# AVAX Fuji Testnet
+
+We are going to test the marketplace on AVAX Fuji Testnet. First, you need to add AVAX Fuji Testnet to metamask. Open metamask and view networks, then click on `Custom RPC`.
+
+![networks](images/networks.png)
```

Comment:
Rename image and image paths to include the tutorial name. Ex `images/networks.png` -> `images/nft-marketplace-networks.png`